### ISSUE TITLE
It seems SentenceTransformer is not able to properly import cutting-edge checkpoints

### ISSUE THREAD

**Issue by daehuikim** [UKPLab/sentence-transformers#3333]
Dear contributors,
While I try to train modernbert with sentencetransforers, I found wired performance close to 0 accuracy.
When I change the model from modernbert into roberta, which is older and less robust one, It shows reasonable performance.
In my opinion, It seems SentenceTransformer is not able to propely import cutting-edge checkpoints like modernbert.
Can anybody investigate this situation?

**Comment by tomaarsen** [comment]
Hello!

Hmm, it surprises me a bit that a ModernBERT-based model does worse than a RoBERTa-based model for you. I've been able to train working ModernBERT models, and so have others in the community. There's currently 216 models: https://huggingface.co/models?library=sentence-transformers&other=modernbert
For example: https://huggingface.co/joe32140/ModernBERT-base-msmarco

The [ModernBERT paper](https://arxiv.org/pdf/2412.13663) in Appendix E.2, Table 9, shows that ModernBERT benefits from a bit of a higher learning rate than RoBERTa - perhaps a higher learning rate helps for you too?

Alternatively, perhaps the two base models that you're considering have already been finetuned differently? E.g. if you're not using https://huggingface.co/answerdotai/ModernBERT-base and https://huggingface.co/FacebookAI/roberta-base, but a finetune instead.

- Tom Aarsen
